funnel/static/translations/hi_IN/messages.json

```diff
@@ -1338,9 +1341,9 @@
         null,
         "<span class=\"mui--hidden-xs mui--hidden-sm\">इस प्रोजेक्ट</span> को सेव न करें"
       ],
```

Comment:
CSS classes should not appear in translations. The source text is incorrectly tagged here.